graphql-parser/src/lexer.rs

```diff
@@ -4,7 +4,7 @@ use std::iter::Peekable;
 
 #[derive(PartialEq, Debug)]
 pub enum Token<'a> {
-  Eof(usize, usize),
+  Eof(usize),
```

Comment:
the issue is on the hi bit, so it does neither. I only removed the lo bit because it made no sense in the eof variant.